_posts/2015-01-05-what-is-color-contrast.md

```diff
@@ -1,22 +1,23 @@
 ---
 layout: post
-title: "What is color contrast?"
+title: "What is Color Contrast?"
 description: "What is color contrast and why do we need it for web accessibility?"
 author: rachel_vasquez
+updated_by: emily_lane
 published: true
 categories:
   - Background
 ---
 
 ## Color Theory
 
-Colors from different segments of the color wheel are contrasting colors. In color theory, [complementary colors](https://en.wikipedia.org/wiki/Complementary_colors) (or "opposite" colors) that are directly across from one another on a basic color wheel provide maximum contrast. This means they stand out from one another the strongest compared to being paired with other colors. Complementary colors are two colors that cancel one another out and produce a white/gray-scale color.
+Contrasting colors, also known as [complementary colors](https://en.wikipedia.org/wiki/Complementary_colors), are colors from opposing segments of the color wheel. Colors that are directly across from one another on a basic color wheel provide maximum contrast.  
```

Comment:
changing complementary colors to contrasting colors is odd... especially since this is under the mantle of 'color theory' still, and in regards to that, "complementary colors" is how these are more often referred to.

Reply:
Hmm, I see where you're coming from. I changed the wording to be more in line with the material of the larger article, since complementary colors are contrasting, and the article is about color contrast. Do you feel that it confuses readers?

Reply:
I get why you did it, it's just awkward in the context of expected terminology when talking about color theory.  Since color theory is just the opening two paragraphs, I'd think it'd be ok to talk about complementary colors in that context, referring to the fact that complementary colors contrast.  And then when the article switches over to "What does this mean for web accessibility?", there's the perfect time to make the switch from the color theory terminology (which designers should be familiar with) to color contrast n' its importance to a11y.

Reply:
I'll rework it a bit, thanks!